sdk/typespec/typespec_client_core/src/http/response.rs

```diff
@@ -64,14 +24,28 @@ impl Model for serde_json::Value {
 /// Given a `Response<T>`, a user can deserialize the body into the intended body type `T` by calling [`Response::into_body`].
 /// However, because the type `T` is just a marker type, the user can also deserialize the body into a different type by calling [`Response::into_json_body`] or [`Response::into_xml_body`],
 /// or access the raw body using [`Response::into_raw_body`].
-pub struct Response<T = ResponseBody> {
+pub struct Response<T = ResponseBody, F = DefaultFormat> {
     status: StatusCode,
     headers: Headers,
     body: ResponseBody,
-    phantom: PhantomData<T>,
+    phantom: PhantomData<(T, F)>,
```

Comment:
Since we wouldn't read this or even need it, I'd preface it with an underscore. That's what I typically do in my code in these cases.

Reply:
I did this, but should note that a `PhantomData` field is _always_ considered live and doesn't require a `_` prefix to avoid the `dead_code` warning.